src/test/java/com/linecorp/armeria/common/stream/PublisherBasedStreamMessageTest.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common.stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.easymock.EasyMock.capture;
```

Comment:
git grep shows more instances of mockito ;)

mockito is the standard for Spring and I guess more heavily used at LINE - I think there was a long period of mockito being quite better than Easymock2, and now that there's Easymock3 there isn't a huge difference, but consistency is king.